19.03-rc/Dockerfile

```diff
@@ -49,5 +49,8 @@ RUN set -eux; \
 COPY modprobe.sh /usr/local/bin/modprobe
 COPY docker-entrypoint.sh /usr/local/bin/
 
+# DOCKER_TLS_CERTDIR is where certificates for TLS will be automatically pulled from (and generated into for dind)
```

Comment:
this comment seems confusing, because /certs needs to be mounted manually into client containers